packages/xo-collection/package.json

```diff
@@ -43,7 +43,7 @@
     "dev": "cross-env NODE_ENV=development babel --watch --source-maps --out-dir=dist/ src/",
     "prebuild": "rimraf dist/",
     "predev": "yarn run prebuild",
-    "prepublishOnly": "yarn run build",
+    "prepare": "yarn run build",
```

Comment:
It is still needed?

Reply:
No, I will remove all unnecessary changes.